backend/prisma/migrations/PHASES.md

```
@@ -0,0 +1,182 @@
+
+| Prefix      | Purpose           | Examples                     |
+| ----------- | ----------------- | ---------------------------- |
+| `core_`     | Foundation tables | init, nullable*\*, unique*\* |
+| `feature_`  | Business features | saved_pools, user_quests     |
+| `optimize_` | Performance       | query_indexes, indexes       |
+| `fix_`      | Bug fixes         | constraint*\*, data*\*       |
+| `refactor_` | Schema changes    | rename*\*, restructure*\*    |
+
+## Statistics
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)